app/models/parser_version.rb

```diff
@@ -10,16 +10,16 @@ class ParserVersion < ActiveResource::Base
   self.element_name = 'version'
 
   def last_harvested_at
-    job = harvest_jobs('finished')[1]
-    job ? job.start_time : nil
-  end
-
-  def harvest_jobs(status = nil)
-    if status == 'finished'
-      HarvestJob.where(parser_id: parser_id, status: status).desc(:start_time)
-    else
-      HarvestJob.where(parser_id: parser_id).desc(:start_time)
-    end
+    last_finished_harvest_job = HarvestJob.desc(:start_time)
```

Comment:
prefer writing a `where` than chaining `desc` and `find_by`

Reply:
Also since this is not a straight forward query can you add a comment please.

Reply:
isn't the variable name good enough to explain what the query does? `last_finished_harvest_job`

The reason I've replaced the `where`, is that we just want to find 1 record, and where return an array :(.
Do you think if we went like this would be better?

```
  def last_harvested_at
    HarvestJob.where(
      parser_id: parser_id,
      status: 'finished',
      environment: { '$ne': 'preview' }
    ).max(:start_time)
  end
```